sympy/stats/crv_types.py

```diff
@@ -127,10 +127,11 @@
 def _(x):
     return any([is_random(i) for i in x])
 
-def rv(symbol, cls, args):
+def rv(symbol, cls, args, **kwargs):
     args = list(map(sympify, args))
     dist = cls(*args)
-    dist.check(*args)
+    if kwargs.pop('check', True):
```

Comment:
```suggestion
    if check:
```

Reply:
Other distributions also use this line. Their checks should be defaulted to True since they are very quick checks. I'll have to change this functionality in `ContinuousRV` itself.

Reply:
I will add the following 2 lines to `ContinuousRV` and others:

```python
# have a default of False while `rv` should have a default of True
kwargs['check'] = kwargs.pop('check', False)
```

Is there a better way of doing this?